package.json

```diff
@@ -4,13 +4,18 @@
   "description": "A Promise-based collection of helper methods for the Canvas LMS API.",
   "main": "index.js",
   "dependencies": {
+    "bluebird": "^3.5.1",
```

Comment:
I'm not sure that `bluebird` is referenced any where in your code - do you still need it to be included?

Reply:
Yeah, I used it in another context, and forgot to remove.. done